sample-app/src/components/DirectAnswer.tsx

```diff
@@ -0,0 +1,57 @@
+import { useAnswersState } from "@yext/answers-headless-react";
+import { FeaturedSnippetDirectAnswer, FieldValueDirectAnswer } from '@yext/answers-headless';
+import renderWithHighlighting from './utils/renderWithHighlighting';
+import classNames from "classnames";
+import '../sass/DirectAnswer.scss';
+
+interface DirectAnswerProps {
+  cssClasses?: {
+    container?: string,
+    title?: string,
+    content?: string,
+    description?: string
+  }
+}
+
+const defaultCSSClasses = {
+  container: 'DirectAnswer',
+  title: 'DirectAnswer__title',
+  content: 'DirectAnswer__content',
+  description: 'DirectAnswer__contentDescription'
+};
+
+function isFeaturedSnippetDirectAnswer(
+  answer: FeaturedSnippetDirectAnswer | FieldValueDirectAnswer
+): answer is FeaturedSnippetDirectAnswer {
+  return 'snippet' in answer;
```

Comment:
Direct Answers have a type field which can be used instead of checking for the snippet: (https://github.com/yext/answers-core/blob/86645df02a26cb10b928b5de8f7cf09c454b0c04/src/transformers/searchservice/createDirectAnswer.ts)

Reply:
oh neat